lib/gooddata/lcm/actions/ensure_technical_users_project.rb

```diff
@@ -36,7 +39,7 @@ class << self
         def call(params)
           client = params.gdc_gd_client
 
-          technical_users = params.technical_user || []
+          technical_users = params.technical_users || params.technical_user || []
```

Comment:
Refers to 'params' more than self (maybe move it to another class?) (FeatureEnvy)